### PR TITLE
feat: add okta auth helper

### PR DESCRIPTION
**Pull request by VVVENIIDIICI**
## Changes
- [x] Add okta authorization helper. 

## How Has This Been Tested?
Unit test and integration test will be added later.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/auth/login_manager.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    function with user id.
+    """
+    @wraps(func)
+    def decorated_view(*args, **kwargs):
+        (is_authorized, error_message_or_uid) = \
+            okta_helper.is_access_request_authorized(request)
+        if is_authorized:
+            return func(error_message_or_uid)
+        else:
+            return make_response(error_message, 403)
```
Comment (yifeili98) [inline src/auth/login_manager.py:21]: `error_message` should be `error_message_or_uid`

File: src/auth/okta_helper.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        Returns:
+            bool, whether the okta access token is valid.
+            str, the user id if exists. Otherwise, return None.
+        """
+
+        jwt_verifier = JWTVerifier(self.okta_configs['issuer'],
+                                   self.okta_configs['client_id'],
+                                   'api://default')
+        try:
+            loop.run_until_complete(jwt_verifier.verify_access_token(token))
```
Comment (yifeili98) [inline src/auth/okta_helper.py:49]: I tried to generate an access token using postman and put the authorization token into the header (followed steps [here](https://developer.okta.com/docs/guides/implement-oauth-for-okta/request-access-token/)). It throws an exception 'No matching JWK' at line 49, did I miss any steps?

Comment (VVVENIIDIICI) [reply src/auth/okta_helper.py:50]: In postman,
URL: `http://127.0.0.1:5000/example_access_token_check` or `http://127.0.0.1:5000/example_id_token_check`
Click `Authorization`, type `Bearer Token`, and then put your token in the token field. Now send the request, and you should not expect an exception.

File: src/auth/login_manager.py (hunk 1 above)
Comment (VVVENIIDIICI) [reply src/auth/login_manager.py]: fixed.

**Review by yifeili98 — APPROVED** [review]
LGTM

**Pull request merged**